src/containers/Navigation/index.tsx

```diff
@@ -1,88 +1,41 @@
-import React, { useState, useRef, useEffect } from 'react';
+import React, { useState } from 'react';
 import NavigationSection from '../../components/NavigationSection';
 import { NavigationSectionData, NavigationSectionItem } from '../../types';
 import { isMobileScreen } from '../../util/isScreenWithinWidth';
-import { scrollTo, calcNavScrollParams } from '../../util/scrollTo';
 
 interface Props {
   sections: NavigationSectionData;
   currentSlug: string;
-  previousSlug: string;
   label: string;
   category: string;
 }
 
 const Navigation = ({
   sections,
   currentSlug,
-  previousSlug,
   label,
   category,
 }: Props): JSX.Element => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
-  const navElement = useRef<HTMLElement | null>(null);
   const toggle = (): void => setIsOpen(!isOpen);
-  const onItemClick = (): void => {
+
+  const onItemClick = () => {
     if (isMobileScreen()) {
       toggle();
     }
   };
 
-  useEffect(() => {
```

Comment:
There's no need to scroll anymore as the Navigation items are togglable now, and this logic was not 100% functional and hacky.